flink-sliding-feature-view/main.py

```diff
@@ -102,6 +101,7 @@
         timestamp_field="timestamp",
         timestamp_format="%Y-%m-%d %H:%M:%S",
         startup_mode="earliest-offset",
+        is_bounded=True
```

Comment:
Would it be better to remove this config and use `to_pandas(force_bounded=True)` to demonstrate the idea that we can switch between production/experiment modes without changing the feature definition?

Reply:
You are correct that we demonstrate the usage of `to_pandas(force_bounded=True)`. However, due to the limitation of Flink, we have to keep the right table to be bounded. Otherwise, an exception will be thrown when a bounded left table joins an unbounded right table.